backend/deepchecks_monitoring/bgtasks/alert_task.py

```diff
@@ -125,7 +125,8 @@ async def execute_monitor(
     ))
 
     if monitor is None:
-        raise ValueError(f'Did not find monitor with the id {monitor_id}')
+        logger.info('Did not find monitor with the id: %s', monitor_id)
```

Comment:
maybe warning? Also printing the organization_id?
Also, how do we reach into a situation that we do not find monitor?

Reply:
it was deleted, no need for a warning imo

Reply:
Info is maybe even too much maybe